riscof/requirements.txt

```diff
@@ -3,4 +3,4 @@ click>=7.1.2
 Jinja2>=2.10.1
 pytz>=2019.1
 riscv-config>=3.2.0
-riscv_isac>=0.7.2
+riscv_isac @ git+https://github.com/riscv-software-src/riscv-isac@777d2b4
```

Comment:
The riscv-isac repo has been archived and the tool is now stored in the riscv-arch-test repo. I've been using the following line in a requirements.txt file with success:
```
riscv-isac @ git+https://github.com/riscv-non-isa/riscv-arch-test/#subdirectory=riscv-isac
```